Queue/Queue.cpp

```diff
@@ -11,26 +11,26 @@ void CreateQueue(Queue& q, int size)
 }
 
 
-int size(Queue& q)
+int GetSize(Queue& q)
```

Comment:
We need to follow a same strategy for naming convention. For example: Upper Camel Case. (https://en.wikipedia.org/wiki/Camel_case)